src/EnergyPlus/DXCoils.cc

```diff
@@ -6489,6 +6489,28 @@ void GetDXCoils(EnergyPlusData &state)
                                     Coil.Name);
             }
 
+            if (Coil.ReportCoolingCoilCrankcasePower) {
+                SetupOutputVariable(state,
+                                    "Cooling Coil Crankcase Heater Electricity Rate",
+                                    OutputProcessor::Unit::W,
+                                    Coil.CrankcaseHeaterPower,
+                                    OutputProcessor::SOVTimeStepType::System,
+                                    OutputProcessor::SOVStoreType::Average,
+                                    Coil.Name);
+                SetupOutputVariable(state,
+                                    "Cooling Coil Crankcase Heater Electricity Energy",
+                                    OutputProcessor::Unit::J,
+                                    Coil.CrankcaseHeaterConsumption,
+                                    OutputProcessor::SOVTimeStepType::System,
+                                    OutputProcessor::SOVStoreType::Summed,
+                                    Coil.Name,
+                                    _,
+                                    "Electricity",
```

Comment:
📌 "Electricity", "DHW", are "Plant" are all enumerated values (ResourceType/FuelType, EndUse, MeterSuperGroup), correct? You know what I am going to say then.

Reply:
Never do today what you can do tomorrow. Something may occur to make you regret your premature action. Aaron Burr.

Reply:
`(lambda f.(lambda x.f(x x) (lambda x.f(x x))) lambda y.("Never do y what you can do y+1"))` --Haskell Curry.

Reply:
```cpp
while (true) {
   if (can_do_x_tomorrow) {
       sleep(24*60*60);
       continue;
   } else {
        do_x();
        break;
}
```